Framework/DataHandling/src/SaveAscii2.cpp

```diff
@@ -72,6 +73,7 @@ void SaveAscii2::init() {
                   "single-spectrum workspaces. "
                   "It is always written for workspaces with multiple spectra, "
                   "unless spectrum axis value is written. Ignored for Table Workspaces.");
+  declareProperty("OneSpectrumPerFile", false, "If true, each spectrum will be saved in an invididual file");
```

Comment:
New properties should be ideally declared at the end. There is also a typo in the doc-line.

Reply:
I moved this to the end and corrected the typo.